dcm4che-net/src/main/java/org/dcm4che3/net/Device.java

```diff
@@ -1164,6 +1147,19 @@ public final TrustManager getTrustManager() {
         return tm;
     }
 
+    private ApplicationEntity getApplicationEntityOrByAETitleAlias(String aet) {
+        ApplicationEntity ae = applicationEntitiesMap.get(aet);
+
+        if (ae == null) {
+            //check alias AE titles
+            return applicationEntitiesMap.values().stream()
+                                         .filter(applicationEntity -> applicationEntity.getAETitleAliases().contains(aet))
```

Comment:
Not sure why IntelliJ keeps doing that, but line continuation should be only double tabbed; shifting code like that just reduces the amount of available space left...remember lines shouldn't be longer than 120 characters. I thought we fixed that?

https://google.github.io/styleguide/javaguide.html